### PR TITLE
backstage: fix component tokens building

### PR DESCRIPTION
**Pull request by frshwtr**
## Describe your changes
Component tokens marked as 'source' in Tokens Studio were not included when building component tokens with style dictionary. Added as `includes` to our style dictionary config.

## Checklist before requesting a review
- [ ] I have applied `percy` label on my PR before merging and after review.
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: apps/dictionary/scripts/build-config/utils.js [hunk 1]
```diff
@@ -309,20 +323,48 @@ export function getBrandSources(brand) {
...
+	const tokenSet = filePath
+		.replace(`${process.cwd()}/tokens`, '')
+		.replace(/^\//, '')
+		.replace('.json', '');
+
+	const theme = getTokensStudioTheme(tokenSet);
+
+	const isTokenStudioSource = theme.selectedTokenSets[tokenSet] === 'source';
+	return isTokenStudioSource;
 }
```
Comment (notlee) [inline apps/dictionary/scripts/build-config/utils.js:370]: This is a little wild and could maybe use some code comments? Looks like it'll do the trick though :)

Comment (notlee) [reply apps/dictionary/scripts/build-config/utils.js:370]: I'm not sure what the best naming is here in terms of brands vs. themes 🤔 

```diff
+       // Check if the token set matches a know brand or sub-brand
+       // `core/components/o3-button.json` - `core/components` is not a sub-brand
+       // `core/professional/ components/o3-button.json` - `core/professional` is a sub-brand
	const subTheme = tokenStudioThemes.find(
		theme => tokenStudioThemeToBrand(theme) === subThemeKey
	);
```

Comment (frshwtr) [reply apps/dictionary/scripts/build-config/utils.js:370]: I've used Tokens Studio naming here. The function name mentions tokens studio as the context and I felt it's best to keep to its terms when working within its namespace.

I do find the misalignment of terms across tools confusing at times however.

Comment (frshwtr) [reply apps/dictionary/scripts/build-config/utils.js:370]: Ah I see your point now though, we've used brand in our other functions.

**Review by notlee — APPROVED** [review]

**Pull request merged**